draft-ietf-add-ddr.md

```diff
@@ -231,13 +231,10 @@ Authenticated Discovery is a mechanism that allows automatic use of a
 Designated Resolver that supports DNS encryption that performs a TLS handshake.
 
 In order to be considered an authenticated Designated Resolver, the
```

Comment:
I don't think that you can consider a resolver to be "authenticated" if all you have is an IP cert for an IP address that you found in an RA/DHCP packet.  That's no better than something that you found on a USB stick in a Walmart car park.

This is a policy statement: a client chooses to use a secured protocol for talking to a discovered resolver on the basis that it couldn't be worse than using an unsecured protocol.  This document claims that ownership over the IP address is strictly better.  That only assumes that the PKI you use to verify that ownership is trustworthy.  Thus, the secured protocol is OK to use.

Reply:
@martinthomson if you look in context, this is in the definition for the "Authenticated mode" vs the "Opportunistic mode". This could also be put as "the mode where you check for the address in a certificate" and "the mode where you require the resolver to be accessed on the same IP address, and check nothing in the certificate".

We can editorially find-and-replace these terms if we want, but that should not be in this PR.